probes/entries.go

```diff
@@ -37,12 +48,22 @@ var (
 		toolPyUpInstalled.Run,
 		toolSonatypeLiftInstalled.Run,
 	}
+	Fuzzing = []ProbeImpl{
+		fuzzedWithOSSFuzz.Run,
+		fuzzedWithOneFuzz.Run,
+		fuzzedWithGoNative.Run,
+		fuzzedWithClusterFuzzLite.Run,
+		fuzzedWithPropertyBasedHaskell.Run,
+		fuzzedWithPropertyBasedTypescript.Run,
```

Comment:
note: we need to change the raw check code to provide more granularity than just "property-based". We need the probes to be at the level of the tool https://github.com/ossf/scorecard/issues/3231